app/src/main/java/com/duckduckgo/app/browser/webview/MaliciousSiteBlockerWebViewIntegration.kt

```diff
@@ -147,35 +153,49 @@ class RealMaliciousSiteBlockerWebViewIntegration @Inject constructor(
 
         val decodedUrl = URLDecoder.decode(url.toString(), "UTF-8").lowercase()
 
-        if (processedUrls.contains(decodedUrl)) {
-            processedUrls.remove(decodedUrl)
-            Timber.tag("PhishingAndMalwareDetector").d("Already intercepted, skipping $decodedUrl")
-            return IsMaliciousViewData.Safe
-        }
-
         val exemptedUrl = exemptedUrlsHolder.exemptedMaliciousUrls.firstOrNull { it.url.toString() == decodedUrl }
 
         if (exemptedUrl != null) {
-            Timber.tag("MaliciousSiteDetector").d("Previously exempted, skipping $decodedUrl as ${exemptedUrl.feed}")
+            Timber.d("Previously exempted, skipping $decodedUrl as ${exemptedUrl.feed}")
```

Comment:
I see we've removed tag, not sure if intentional

Reply:
Yes, similar to https://github.com/duckduckgo/Android/pull/5604#discussion_r1977087683, we don't use tag for other logs in this class